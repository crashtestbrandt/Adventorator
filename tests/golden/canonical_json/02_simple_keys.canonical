{"a":2,"m":3,"z":1}