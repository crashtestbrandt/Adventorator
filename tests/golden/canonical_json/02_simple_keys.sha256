ebba85cfdc0a724b6cc327ecc545faeb38b9fe02eca603b430eb872f5cf75370
