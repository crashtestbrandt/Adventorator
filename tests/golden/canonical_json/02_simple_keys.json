{
  "z": 1,
  "a": 2,
  "m": 3
}